cl/assets/templates/cotton/navigation_menu.html

```diff
@@ -0,0 +1,155 @@
+{% load svg_tags static component_tags %}
+{% require_script "js/alpine/components/navigation_menu.js" %}
+{% require_script "js/alpine/components/intersect.js" %}
```

Comment:
The import path for the `intersect.js` file is incorrect.

```suggestion
{% require_script "js/alpine/composables/intersect.js" %}
```